src/core/psxmem.cc

```diff
@@ -274,6 +325,43 @@ uint32_t PCSX::Memory::read32(uint32_t address) {
     }
 }
 
+int PCSX::Memory::sendReadToLua(const uint32_t address, const size_t size) {
+    // Grab a local pointer for our Lua VM interpreter
+    auto L = *g_emulator->m_lua;
+    int nresult = 0;
+    // Try getting the symbol 'UnknownMemoryRead' from the global space, and put it on top of the stack
+    L.getfield("UnknownMemoryRead", LUA_GLOBALSINDEX);
+    // If top-of-stack is not nil, then we're going to try calling it.
+    if (!L.isnil()) {
+        try {
+            // Call the function. Technically we should empty
+            // the stack afterward to clean potential return
+            // values from the Lua code, this is a bug in the gui code.
+            const int top = L.gettop();
+            L.push(lua_Number(address));
+            L.push(lua_Number(size));
+            nresult = L.pcall(2);
+        } catch (...) {
+            // If there is any error while executing the Lua code,
+            // push the string "UnknownMemoryRead" on top of the stack,
+            L.push("UnknownMemoryRead");
+            // then push the value "nil" (rough equivalent of NULL in Lua)
+            L.push();
+            // This will pop a key/value pair from the stack, and set
+            // the corresponding value in the global space. This effectively
+            // deletes the global function on errors.
+            L.settable(LUA_GLOBALSINDEX);
+        }
+    } else {
+        // This pops the nil from the stack.
+        L.pop();
+    }
+
+    return nresult;
+}
+
+void PCSX::Memory::sendWriteToLua() {}
```

Comment:
This looks defined but not used ?

Reply:
Woops. I had removed this but it came back after I had that commit/push bring back everything and had to re-force push.